testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/http/AbstractHttpClientTest.java

```diff
@@ -1029,6 +1029,34 @@ void spanEndsAfterHeadersReceived() throws Exception {
         });
   }
 
+  @Test
+  void requestClientError() throws Exception {
```

Comment:
this is essentially a copy of `errorSpan` test, perhaps it would make sense to parameterize that test instead

Reply:
done